x/emissions/module/network_summaries.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // Implements function phi prime from litepaper
 // φ'_p(x) = p * (ln(1 + e^x))^(p-1) * e^x / (1 + e^x)
-func gradient(p float64, x float64) (float64, error) {
```

Comment:
doesnt need to be exported if the tests belong in the same package. unless of course this is used outside of module somewhere.